openassessment/assessment/test/test_staff.py

```diff
@@ -1,13 +1,15 @@
 # coding=utf-8
+"""
+Tests for staff assessments.
+"""
```

Comment:
The number of lines in this diff view has changed significantly, as I was getting a frustrating quality failure on travis. To fix it, I went through and manually fixed all the quality errors in this file, and lowered the error threshold to match.